front/lib/api/config.ts

```diff
@@ -147,7 +147,7 @@ const config = {
   getOAuthNotionClientId: (): string => {
     return EnvironmentConfig.getEnvVariable("OAUTH_NOTION_CLIENT_ID");
   },
-  getOAuthNotionPlatformActionsClientId: (): string => {
+  getOAuthNotionToolsClientId: (): string => {
```

Comment:
For consistency I think you should keep it as is was before, see for example Github just before. 
We use `PLATFORM_ACTIONS` in a lot of places 🙏🏻